docker-compose.yaml

```diff
@@ -4,7 +4,6 @@ services:
   mosquitto:
     container_name: mosquitto
     image: eclipse-mosquitto:2.0.14
-    user: ${USER_ID}:${GROUP_ID}
```

Comment:
this was needed for the other project, but here is not an issue, for our purposes